### PR TITLE
Add a version-check to BigInt quick-eval test

### PR DESCRIPTION
**Pull request by d10c**
Closes https://github.com/github/vscode-codeql/issues/3759

Fixes a problem in https://github.com/github/vscode-codeql/pull/3647 where the tests did not pass under older CLI versions.

This separates the BigInt code into its own file and quick-evals it only on CLI versions that support `QlBuiltins::BigInt`.

### CODE REVIEW

**Comment by d10c** [comment]
Tests pass on older CLI versions: https://github.com/github/vscode-codeql/actions/runs/11343913756 

This PR is ready to merge pending approval.

**Review by koesie10 — APPROVED** [review]
Thanks, LGTM!

**Pull request merged**
